src/rpc/core_rpc_server.cpp

```diff
@@ -198,7 +199,9 @@ namespace cryptonote
     res.start_time = (uint64_t)m_core.get_start_time();
     res.free_space = m_restricted ? std::numeric_limits<uint64_t>::max() : m_core.get_free_space();
     res.offline = m_core.offline();
+    res.bootstrap_daemon_address = m_bootstrap_daemon_address;
     res.height_without_bootstrap = res.height;
+    res.was_bootstrap_ever_used = m_was_bootstrap_ever_used;
```

Comment:
You need to lock the mutex here too as you're reading from `m_was_bootstrap_ever_used`.